packages/rich-text/src/to-dom.js

```diff
@@ -122,7 +122,6 @@ function prepareFormats( prepareEditableTree = [], value ) {
 export function toDom( {
 	value,
 	multilineTag,
-	multilineWrapperTags,
```

Comment:
We still pass `multilineWrapperTags` to this function here:

https://github.com/WordPress/gutenberg/blob/2889c29458489876a941cf4aaae093b10c1990e8/packages/block-editor/src/components/rich-text/index.js#L941-L946

Can we expect that to continue working as expected?

Reply:
Also here:

https://github.com/WordPress/gutenberg/blob/2889c29458489876a941cf4aaae093b10c1990e8/packages/rich-text/src/test/to-dom.js#L24-L41

Reply:
Thanks for catching! Removed in abde860.